resources/gosh/README.md

```diff
@@ -0,0 +1,11 @@
+# `@keymanapp/resources-gosh`
+
+> TODO: description
+
+## Usage
+
+```
+const resourcesGosh = require('@keymanapp/resources-gosh');
+
+// TODO: DEMONSTRATE API
```

Comment:
Not sure we really need this given intended use is probably always outside js code?

Reply:
Sorry, the entire README was created by `lerna create` which generates this boilerplate. Since this is a draft, I never modified the README. I'll change it!